### DISCUSSION TITLE
Pycharm Myst-Markdown Plugin

### DISCUSSION THREAD

**Discussion by MordicusEtCubitus** [executablebooks/meta#798] [Q&A]
Hi,
First thanks for the nice jupyter-book suite, this is huge !

Do you know if there is a Myst-Markdown plugin for PyCharm ?
I've tried the extension in jupyterlab and vscode myst extension that are great, but I am not able to find a myst extension for PyCharm.

Thanks for your answer

**Comment by choldgraf** [comment]
I think that @pauleveritt is probably in the best position to know if there is MyST support in pycharm. I don't know of any myself, but perhaps the work on formalizing the myst spec will make this more straightforward to do.

**Reply by pauleveritt** [reply]
Alas, no special support beyond general Markdown.

**Comment by andylinzie** [comment]
For anyone visiting this issue, this is now a thing; see [here](https://youtrack.jetbrains.com/issue/PY-61291/Support-MyST-syntax).
